ToolkitComponent.SourceProject.props

```diff
@@ -3,18 +3,23 @@
     <Error Condition="$(ToolkitComponentName) == ''" Message="ToolkitComponentName is not defined. Please check your csproj." />
   </Target>
 
-  <!-- Set up the MultiTarget system -->
-  <Import Project="$(ToolingDirectory)\MultiTarget\Library.props" />
-
   <PropertyGroup>
     <DateForVersion Condition="'$(DateForVersion)' == ''">$([System.DateTime]::UtcNow.ToString(yyMMdd))</DateForVersion>
     <Version Condition="'$(Version)' == ''">$(MajorVersion).$(MinorVersion).$(DateForVersion)</Version>
     <Version Condition="'$(PreviewVersion)' != ''">$(Version)-$(PreviewVersion)</Version>
+  </PropertyGroup>
+
+  <!-- Set up the MultiTarget system -->
+  <Import Project="$(ToolingDirectory)\MultiTarget\Library.props" />
```

Comment:
This used to just be at the top of the property group above, should we just put it back there? Does it need to be between the two groups?

The main change is we just need the SDK extras to happen after these properties get set outside the library.props, right?

Reply:
I had seen issues when `Version` wasn't defined before the `Library.props` import, but testing locally I'm not seeing any immediate issues. I'll make the change and see what the CI catches.